src/rocksdb.rs

```diff
@@ -1133,6 +1164,35 @@ impl Writable for WriteBatch {
             Ok(())
         }
     }
+
+    fn delete_range_cf(&self,
+                       cf: &CFHandle,
+                       begin_key: &[u8],
+                       end_key: &[u8])
+                       -> Result<(), String> {
+        unsafe {
+            crocksdb_ffi::crocksdb_writebatch_delete_range_cf(self.inner,
+                                                              cf.inner,
+                                                              begin_key.as_ptr(),
+                                                              begin_key.len(),
+                                                              end_key.as_ptr(),
+                                                              end_key.len());
+            Ok(())
+        }
+    }
+}
+
+impl WriteBatch {
```

Comment:
Because `delete_range` is not part of `Writable` interface(since it's included for `DB`).